kuma/settings/common.py

```diff
@@ -1507,13 +1507,14 @@ def parse_iframe_url(url):
         1000,
         "Number of expired sessions to cleanup up in one go.",
     ),
-    WELCOME_EMAIL_FROM=(
-        "Janet Swisher <no-reply@mozilla.org>",
-        "Email address from which welcome emails will be sent",
-    ),
     AKISMET_KEY=("", "API key for Akismet spam checks, leave empty to disable"),
 )
 
+# Email address from which welcome emails will be sent
+WELCOME_EMAIL_FROM = config(
+    "WELCOME_EMAIL_FROM", default="Janet Swisher <no-reply@mozilla.org>"
+)
+
```

Comment:
So, it's curious how this ended up there. Should I blame a rogue `git rebase` or a bug in GitHub.com??